skuba-update/skuba_update/skuba_update.py

```diff
@@ -289,19 +290,22 @@ def run_zypper_command(command, needsOutput=False):
     Run the given zypper command. The command is expected to be a tuple which
     also contains the 'zypper' string. It returns the exit code from zypper.
```

Comment:
Missed changing this before making the PR.  Refactored code no longer requires including 'zypper', which makes the calls a little shorter.